### ISSUE TITLE
error with borg 1.0

### ISSUE THREAD

**Issue by lluis** [asifbacchus/MailcowBackup#7]
Hello! I get this error:
```
[2021-02-09 23:35:05] -- [INFO] Executing borg backup operation --
usage: borg [-h] [-V] <command> ...
borg: error: unrecognized arguments: --show-rc
[2021-02-09 23:35:06] -- [ERROR] 138: Borg exited with a critical error. Please review this log file for details. --
```
I think its because I'm using borg 1.0
changing the order of arguments in borgCMD seems to work:

```
diff --git a/backup/backup.sh b/backup/backup.sh
index 1ec685a..2c3c27b 100755
--- a/backup/backup.sh
+++ b/backup/backup.sh
@@ -672,7 +672,7 @@ printf "%s[%s] -- [INFO] Pre-backup tasks completed, calling borgbackup --%s\n"
 ## construct the proper borg commandline
 # base command
 if [ "$exclusions" -eq 0 ]; then
-    borgCMD="borg --show-rc create ${borgCreateParams} \
+    borgCMD="borg create --show-rc ${borgCreateParams} \
         ::$(date +%Y-%m-%d_%H%M%S) \
         ${mcConfig%/*} \
         ${sqlDumpDir} \
@@ -683,7 +683,7 @@ if [ "$exclusions" -eq 0 ]; then
         ${dockerVolumeCrypt} \
         ${xtraList}"
 elif [ "$exclusions" -eq 1 ]; then
-    borgCMD="borg --show-rc create ${borgCreateParams} \
+    borgCMD="borg create --show-rc ${borgCreateParams} \
         --exclude-from ${borgExcludeListPath} \
         ::$(date +%Y-%m-%d_%H%M%S) \
         ${mcConfig%/*} \
```

**Comment by asifbacchus** [comment]
That's odd. I find the documentation a little confusing on this issue actually. In the examples, it always shows 'common options' to the left of the main command like `borg --show-rc create ...`. However, the actual wording of the documentation says all options should be to the left of positional arguments which would imply `borg create --show-rc ...`.

I'll try out the changed order using the latest version and, if it works, I'll update the script. Personally, I prefer the way you've done it.

**Comment by asifbacchus** [comment]
Seems to work and actually the documentation supports this change. Nice catch, thanks! I've updated the script in the version 3.1 release.

closed by f2231c70747124150c9fb920696d26338c81b00b

**Comment by lluis** [comment]
thanks! :)
